### PR TITLE
[Modular] Loads of cosmetics (Nice)

### PR DESCRIPTION
**Pull request by ReturnToZender**
## About The Pull Request
Adds donor sprites, server reward sprites, and ordered sprites from various players.

Everything here has been tested and completed, and is ready for testmerging :)
What needs to be done:
--Add Farren's owl sprites
--add Grim's snout
--add Ion's formal atmos jumpsuit (with all sides) and put it in the correct vendor
--add a CE radkini

## Why It's Good For The Game
This adds a huge amount of extra cosmetics to the loadout menu, and will soon add them to the game in various forms, allowing for more personal expression.

## Changelog

:cl:
add: 2 unauthorized command uniforms (to loadout and command vendor)
add: golden pinwheel hat for people who have referred at least 2 people to Bubberstation (ckey whitelist lists all people so far and will be updated)
add: Korve and Union cards (reskinned cardboard cards, they provide no benefit) for additional customization
add: green resprites of every major medical tool using advanced reskins
add: working runner jackets (they were improperly indented before), trucker jacket, ports Cyberpunk 2077 Ronin jacket from Citadel
add: starwire wine bottle that contains wine, bottle of wine to loadout
add: Glow shoes, mech pilot costume to loadout, 4 plushies to loadout, one works as a slow bar of soap (but cannot slip)
del: redundant _plush in icon names, moves sprites to the proper file
del: two redundant Syndicate turtlenecks, these are already available in proper balanced form in modular_skyrat's loadout_datum_under.dm folder.
fix: loadout jumpsuits for a randomhyena
/:cl:

### CODE REVIEW

**Comment by ReturnToZender** [comment]
Had to make a new version after I renamed my branch

**Comment by BrandonSmith6** [comment]
**Patiently waiting for merge...**

File: modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_pocket.dm [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/datum/loadout_item/pocket_items/wine //Beer is already available, so why not wine? Stay classy while drinking on the job.
+	name = "wine bottle"
+	item_path = /obj/item/reagent_containers/cup/glass/bottle/wine
+
+/datum/loadout_item/pocket_items/starwine //sprites by Keila
+	name = "Swtarwire wine bottle"
```
Comment (StrangeWeirdKitten) [inline modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_pocket.dm:6]:
```suggestion
	name = "swtarwire wine bottle"
```

File: modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_under.dm [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-	donator_only = TRUE
-
-/datum/loadout_item/under/syndicate/skyrat/tactical/skirt
-	name = "tactical skirtleneck"
-	item_path = /obj/item/clothing/under/syndicate/skyrat/tactical/skirt
-	donator_only = TRUE
-
-/datum/loadout_item/under/rank/security/officer/hecu
+/datum/loadout_item/under/jumpsuit/security/hecu
 	name = "Urban camouflage BDU"
```
Comment (StrangeWeirdKitten) [inline modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_under.dm:2]:
```suggestion
	name = "urban camouflage BDU"
```

File: modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_under.dm [hunk 3]
```diff
@@ -29,3 +29,6 @@
 	item_path = /obj/item/clothing/under/misc/maid/tactical
 	donator_only = TRUE
 
+/datum/loadout_item/under/miscellaneous/mech_suit
+	name = "Mech pilot's suit"
```
Comment (StrangeWeirdKitten) [inline modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_under.dm:33]:
```suggestion
	name = "mech pilot's suit"
```

**Review by StrangeWeirdKitten — CHANGES_REQUESTED** [review]

**Comment by StrangeWeirdKitten** [comment]
Test merging

**Pull request merged**